app/models/contributer.rb

```diff
@@ -1,4 +1,6 @@
 class Contributer < CDQManagedObject
```

Comment:
yeah that was my fault, I was pretty sure after it was merged I spelled it wrong :|